lfx-mentorship/2022/01-Spring/README.md

```diff
@@ -414,7 +414,7 @@ This internship involves improving the existing Katacoda scenario and adding new
 
 - Description: Thanos is a distributed system that has a user interface written in React. Let's create a community instance with continuous integration for easy testing of how Thanos works. Also, it could serve as a testing ground for new React components. A server is provided by CNCF (<https://github.com/cncf/cluster/issues/190>).
 - Recommended Skills: Linux, Ansible, Python, Shell Scripting
-- Mentor(-s): Giedrius Statkevičius (@GiedriusS)
+- Mentor(-s): Giedrius Statkevičius (@GiedriusS), Wiard van Rji (@wiardvanrij)
```

Comment:
Please let Wiard to apply as a mentor to the project directly (as suggested here - https://docs.linuxfoundation.org/lfx/mentorship/mentors/create-a-mentor-profile/request-to-be-added-to-a-program).

Reply:
ACK. 

Should I remove this from this PR then?

Reply:
@bwplotka no, please keep this in the PR and let them apply to the project.

Reply:
Will do, but please fix my name ^^

Reply:
Corrected in [d39ef6e](https://github.com/cncf/mentoring/commit/d39ef6e53c3cd75ae2a2ddb2b3eb0431beef6c61)

Reply:
@idvoretskyi  I have applied to the project. Is there something else I need to do? 

Thanks!

Reply:
@wiardvanrij I don’t see your application yet.

Reply:
@wiardvanrij invited, please let me know is any issues!